File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
    bool answer = true;
    
    int p = 0;
    int y = 0;
    
    for (char ch : s) {
        if (ch == 'p' || ch == 'P') p++;
        else if (ch == 'y' || ch == 'Y') y++;
    }

    return p == y;
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s) {
    var p = 0, y = 0;
    
    for (let ch of s) {
        if (ch == 'p' || ch == 'P') p++;
        else if (ch == 'y' || ch == 'Y') y++;
    }

    return (p == y);
}